samples/net/zperf/src/zperf_shell.c

```diff
@@ -218,6 +218,36 @@ int zperf_get_ipv4_addr(const struct shell *shell, char *host,
 	return 0;
 }
 
+const struct in_addr *zperf_get_default_if_in4_addr(void)
+{
+#if CONFIG_NET_IPV4
+	const struct in_addr *in4_addr;
+	const struct in_addr *dst_in4_addr;
+
+	dst_in4_addr = net_ipv4_unspecified_address();
```

Comment:
We do not need dst_in4_addr variable, just use the net_ipv4_unspecified_address() directly in next line.
Also in4_addr is useless, just do
```
return net_if_ipv4_select_src_addr(NULL, net_ipv4_unspecified_address());
```
here.